src/Symfony/Bundle/SecurityBundle/composer.json

```diff
@@ -20,6 +20,7 @@
         "ext-xml": "*",
         "symfony/config": "^4.2|^5.0",
         "symfony/dependency-injection": "^4.2|^5.0",
+        "symfony/event-dispatcher": "^4.4|^5.0",
```

Comment:
not sure this is needed, event-dispatcher is already a dependency of http-kernel and that won't change
should be removed to me

Reply:
hmm, that's to get the pass, because HK allows v4.3 (but it doesn't allow v5 :) )
what about bumping ED to 4.4 in HK instead?

Reply:
Since the bundle directly uses a class from that package, I thought having a direct dependency would be justified. Does Symfony have some kind of policy when to add direct dependencies and when to rely on transtitive dependencies?

Reply:
the policy is that the code should have tests, and tests should pass :)

Reply:
All right. I've reverted the line and bumped event-dispatcher in http-kernel instead. And I've added a test covering the SecurityBundle aliases. 🙃